script.js

```diff
@@ -1,4 +1,122 @@
+/*Base
+Requirements
+There are 2 players and players take turns.
+When a player clicks Submit, the game rolls 2 dice and shows the dice rolls, for example 3 and 6.
+The player picks the order of the dice they want. For example, if they wanted the number 63, they would specify that the 2nd dice goes first. You can choose how the player specifies dice order.
+After both players have rolled and chosen dice order, the player with the higher combined number wins.*/
+
+/*
+Welcome Player 1.
+You rolled 3 for Dice 1 and 6 for Dice 2.
+Choose the order of the dice.
+
+Player 1, you chose Dice 2 first.
+Your number is 63.
+It is now Player 2's turn.
+*/
+
+//game mode
+var modeDiceRoll = "Start";
+var modeDiceOrder = "Userinput";
+var modeGameWinnner = "Winner";
+//default game mode
+var gameMode = modeDiceRoll;
+// players
+var playerOne = "Player One";
+var playerTwo = "Player Two";
+// arrays to store dice rolls
+var playerOneDice = [];
+var playerTwoDice = [];
+// player dice order
+var playerOneOrder;
+var playerTwoOrder;
+//win counter
+var playerOneWin = 0;
+var playerTwoWin = 0;
+// random dice roll function
+var generateRandomDice = function () {
+  var randomDiceRoll = Math.floor(Math.random() * 6) + 1;
+  return randomDiceRoll;
+};
+// default player = 1
+var currentPlayer = playerOne;
+
+var storeDiceRolls = function () {
+  var diceRolls = [generateRandomDice(), generateRandomDice()];
+
+  if (currentPlayer == playerOne) {
+    playerOneDice = diceRolls;
+    console.log("player1 " + diceRolls);
+  } else if (currentPlayer == playerTwo) {
+    playerTwoDice = diceRolls;
+    console.log("player2 " + diceRolls);
+  }
+  return diceRolls;
+};
```

Comment:
We usually don't assign arrays like this. It's better to do the push method 
Reason being this idea of reference vs primitive types. - I'll run through this in class ! 


[Watch this video to find out more :) ](https://www.youtube.com/watch?v=9ooYYRLdg_g)